importer.js

```diff
@@ -38,143 +35,117 @@ function flatSchema(schema) {
+      this.fieldSchema = fieldsFlatMap.schema;
+      this.fieldsNamesMapping = fieldsFlatMap.namesMapping;
+      this.tagSchema = tagsFlatMap.schema;
+      this.tagsNamesMapping = tagsFlatMap.namesMapping;
 
-    client.schema(this.config.measurementName, this.fieldSchema, this.tagSchema, {
-      // default is false
-      stripUnknown: true,
-    });
+      client.schema(this.config.measurementName, this.fieldSchema, this.tagSchema, {
+        // default is false
+        stripUnknown: true,
+      });
 
-    // callback for checking columns names in csv
+      // callback for checking columns names in csv
 
-    this.config.csv.columns = (cols) => {
-      for(var key in this.fieldSchema) {
-        // if 'from' field is an array - checking each of array items
-        if(Array.isArray(this.fieldsNamesMapping[key])) {
-          this.fieldsNamesMapping[key].forEach(el => this._checkColInCols(el, cols));
-        } else {
-          this._checkColInCols(this.fieldsNamesMapping[key], cols);
+      this.config.csv.columns = (cols) => {
+        for(var key in this.fieldSchema) {
+          // if 'from' field is an array - checking each of array items
+          if(Array.isArray(this.fieldsNamesMapping[key])) {
+            this.fieldsNamesMapping[key].forEach(el => this._checkColInCols(el, cols));
+          } else {
+            this._checkColInCols(this.fieldsNamesMapping[key], cols);
+          }
         }
-      }
 
-      // callback should return list of columns' names
-      return cols;
-    };
-
-    var parser = parse(this.config.csv);
-    var input = fs.createReadStream(this.inputFile);
-
-    console.log('Importing');
-    var num = 0;
-
-    var transformer = transform((record, callback) => {
-      num++;
-      // TODO: add filter
-      this._writeRecordToInflux(record)
-        .then(() => {
-          if(this.isQuiteMode) {
-            this.progressBar.tick();
-          }
-          callback(null, '');
-        })
-        .catch(err => {
-          console.error('\n [' + num + '] BAD_WRITE');
-          console.error(record);
-          console.error(err);
-          console.error(JSON.stringify(err, null, 2));
-          if(this.isQuiteMode) {
-            this.progressBar.tick();
-          }
-          //process.exit(errors.ERROR_BAD_WRITE);
-          callback(null, '');
-        });
-    }, { parallel: 1 });
-
-    input
-      .pipe(parser)
-      .pipe(transformer)
-      .pipe(process.stdout);
+        // callback should return list of columns' names
+        return cols;
+      };
+
+      var parser = parse(this.config.csv);
+      var input = fs.createReadStream(this.inputFile);
+
+      console.log('Importing');
+      var num = 0;
+
+      var transformer = transform((record, callback) => {
+        num++;
+        // TODO: add filter
+        this._writeRecordToInflux(record)
+          .then(() => {
+            if(this.isQuiteMode) {
+              this.progressBar.tick();
```

Comment:
this method would be better to use in index.js cuz Importer should don't know about console things. 
But lets keep it here for this PR